plugins/extra/runescape.go

```diff
@@ -118,115 +120,166 @@ func getCombatLevel(attack, defence, strength, hitpoints, ranged, prayer, magic
 	return int(math.Floor(base + math.Max(meleeOption, math.Max(rangedOption, magicOption))))
 }
 
-func (p *runescapePlugin) getPlayerSkills(search string) (runescapeLevelMetadata, error) {
-	var emptySkill runescapeLevelMetadata
+func (p *runescapePlugin) getPlayerSkills(search string) (map[string]runescapeLevelMetadata, error) {
+	var emptySkills map[string]runescapeLevelMetadata
 
 	found := false
 	player := ""
-	skill := ""
+	skillsString := ""
+	var skills []string
 
 	matches := levelRegex.FindAllStringSubmatch(search, -1)
 	for _, v := range matches {
 		if strings.HasPrefix(v[1], "\"") {
 			v[1] = v[1][1 : len(v[1])-1]
 		}
 		player = v[1]
-		skill = v[2]
+		skillsString = v[2]
+		skills = whitespaceRegex.Split(skillsString, -1)
```

Comment:
How does this handle multiple spaces? I think I've used string.Fields for stuff like this in the past

Reply:
The regex is `\s+` so it should match multiple spaces of any kind but I'll switch it.